mep/books/templates/books/snippets/item_result.html

```diff
@@ -1,49 +1,71 @@
 {% load static %}
-<li class="books result">
-    <a href="#" aria-label="{{ item.title }}">
-        <article>
-            <div class="main">
-                <span class="counter"></span>
-                <div class="metadata">
-                    <h1 class="title">{{ item.title }}</h1>
-                    {% if item.authors %}
-                    <ul class="authors">
-                        {% for author in item.authors %}
-                            {% if forloop.counter <= 3 %}
-                            <li class="author">{{ author }}</li>
-                            {% endif %}
-                        {% endfor %}
-                        {% if item.authors|length > 3 %}
-                            <span>... {{ item.authors|length|add:"-3" }} more authors</span>
-                        {% endif %}
-                    </ul>
-                    {% endif %}
-                    {% if item.pub_date %}
-                    <span class="dates">
-                        <time datetime="{{ item.pub_date }}">{{ item.pub_date }}</time>
-                    </span>
-                    {% endif %}
-                </div>
-            </div>
-            <div class="secondary">
-                {% if item.publishers %}
-                <h2 class="publisher-label">Publishers</h2>
-                <ul class="publishers">
-                    {% for publisher in item.publishers %}
-                        {% if forloop.counter <= 3 %}
-                        <li class="publisher">{{ publisher }}</li>
-                        {% endif %}
-                    {% endfor %}
-                    {% if item.publishers|length > 3 %}
-                        <span>... {{ item.publishers|length|add:"-3" }} more publishers</span>
-                    {% endif %}
-                </ul>
+<dl class="book">
+    <div class="primary">
+        {# title #}
+        <div class="title">
+            <dt>title</dt>
+            <dd>{{ item.title }}</dt>
+        </div>
+        {# authors #}
+        {% if item.authors|length > 0 %}
+        <div class="authors">
+            <dt>authors</dt>
+            {% for author in item.authors %}
+                {% if forloop.counter <= 3 %}
+                <dd>{{ author }}</dd>
                 {% endif %}
-            </div>
-            <span class="see-more">
-                See more details
-                <img class="triangle icon" src="{% static 'img/icons/more_triangle.svg' %}" alt="">
-            </span>
-        </article>
-    </a>
-</li>
\ No newline at end of file
+            {% endfor %}
+            {% if item.authors|length > 3 %}
+            <dd>... {{ item.authors|length|add:"-3" }} more authors</dd>
+            {% endif %}
+        </div>
+        {% endif %}
+        {# editors #}
+        {% if item.editors.count > 0 %}
```

Comment:
Is this because we only show editors, translators etc. on detail page (which is backed by db record) and not on search results (backed by solr result)? That definitely warrants a template comment so we're explicit about what data we expect where.